TypeCobol.Test/Parser/Programs/Cobol85/ReferenceModifiers.rdz.cbl

```diff
@@ -0,0 +1,64 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 item   PIC X(20).
+       01 part   PIC X(20).
+       01 var0   PIC 9 VALUE 0.
+       01 var1   PIC 9 VALUE 1.
+      
+       PROCEDURE DIVISION.
+      * OK this is a shorthand
+           MOVE item(2:) TO part
+      * KO zero position
+           MOVE item(0:) TO part
+      * KO negative position
+           MOVE item(-1:) TO part
+      * KO by a previous step
+      *     MOVE item(:2) TO part
+      
+      * KO zero position
+           MOVE item(0:5) TO part
+      * KO zero length
+           MOVE item(1:0) TO part
+      * KO*2 zero values
+           MOVE item(0:0) TO part
+      * KO negative position
+           MOVE item(-1:5) TO part
+      * KO negative length
+           MOVE item(1:-5) TO part
+      * KO*2 negative values
+           MOVE item(-1:-5) TO part
+      * OK
+           MOVE item(1.32:5) TO part
```

Comment:
I wonder if we should create a warning for those borderline uses. IBM compiler does not, but...

Reply:
I think this could be a quality rule, but should not be handled by the parser.